newIDE/app/src/MainFrame/index.js

```diff
@@ -1238,6 +1243,18 @@ const MainFrame = (props: Props) => {
     toolbar.current.setEditorToolbar(editorToolbar);
   };
 
+  const hideEditorToolbar = (hidden: boolean) => {
+    if (!toolbar.current) return;
+
+    toolbar.current.hideToolbar(hidden);
+  };
+
+  const hideTabsTitlebar = (hidden: boolean) => {
```

Comment:
Could we have just a single function that does both thing? Seems like we also want both or none.

Reply:
True, at the time it felt more convenient to be able to handle both separately, but maybe not. I'll factor this